FusionIIIT/applications/department/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.conf.urls import url
+
+from . import views
+
+app_name = 'dep'
+
+urlpatterns = [
+
+    url(r'^$', views.hod, name='dep'),
+    url(r'^file_request/$', views.file_request, name='file_request'),
+    url(r'^BtechFirstYear_Students/$', views.BtechFirstYear_Students, name='BtechFirstYear_Students'),
+    url(r'^BtechSecondYear_Students/$', views.BtechSecondYear_Students, name='BtechSecondYear_Students'),
+    url(r'^BtechThirdYear_Students/$', views.BtechThirdYear_Students, name='BtechThirdYear_Students'),
+    url(r'^BtechFinalYear_Students/$', views.BtechFinalYear_Students, name='BtechFinalYear_Students'),
+<<<<<<< HEAD
```

Comment:
Don't push the code that can break the application. Resolve the conflicts properly. VS Code has an excellent feature to resolve conflict. Make sure you do this before committing your code.